packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1182,6 +1182,7 @@ export const UnknownCLIError = {
 } satisfies ErrorData;
 /**
  * @docs
+ * @message `astro sync` command failed to generate content collection types.
  * @description
  * `astro sync` command failed to generate content collection types.
```

Comment:
Not blocking, but this description is kinda terrible

Reply:
Now that you bring attention here, it is odd that a syntax error in markdown is one of the reasons, considering that you mentioned there are two different concerns here.